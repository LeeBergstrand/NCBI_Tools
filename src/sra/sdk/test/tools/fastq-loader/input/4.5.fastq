@BILLIEHOLIDAY_1_FC20F3DAAXX:8:2:342:540/1
G1231231231231231 2312312312312312312
@BILLIEHOLIDAY_1_FC20F3DAAXX:8:2:342:540/1
G12112312312312312 312312312312312312
@BILLIEHOLIDAY_1_FC20F3DAAXX:8:2:342:540/1
G123223123123123123 12312312312312312
@BILLIEHOLIDAY_1_FC20F3DAAXX:8:2:342:540/1
G1231331231231231231 2312312312312312
@BILLIEHOLIDAY_1_FC20F3DAAXX:8:2:342:540/1
G12312112312312312312 312312312312312
