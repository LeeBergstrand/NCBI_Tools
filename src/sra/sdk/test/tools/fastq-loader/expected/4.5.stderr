latf-load err: file="4.5.fastq"
latf-load err:syntax error
latf-load err: file="4.5.fastq"
latf-load err:syntax error
latf-load err: file="4.5.fastq"
latf-load err:syntax error
latf-load err: error excessive while reading file within alignment module - Number of errors 3 exceeds limit of 2: Exiting
latf-load err: error excessive while reading file within alignment module - accession="obj" errors="3" status="failure"
latf-load err: error excessive while reading file within alignment module - load failed
